src/string.cr

```diff
@@ -4112,11 +4120,13 @@ class String
     io << '}' if char.ord > 0xFFFF
   end
 
+  # Returns true if this string starts with a given *str*.
```

Comment:
```suggestion
  # Returns `true` if this string starts with a given *str*, otherwise `false`.
```

ditto all below

Reply:
```suggestion
  # Returns `true` if this string starts with the given *str*, otherwise `false`.
```
"the" instead of "a" like for the others.